nucliadb_core/src/metrics/metric/tokio_runtime.rs

```diff
@@ -0,0 +1,444 @@
+            "nucliadb_node_total_steal_operations",
+            "The number of times worker threads stole tasks from another worker thread. [...]",
+            total_steal_operations.clone(),
+        );
+
+        let max_steal_operations = Gauge::default();
+        registry.register(
+            "nucliadb_node_max_steal_operations",
+            "The maximum number of times worker any thread stole tasks from another worker thread",
+            max_steal_operations.clone(),
+        );
+
+        let min_steal_operations = Gauge::default();
+        registry.register(
+            "nucliadb_node_min_steal_operations",
+            "The minimum number of times worker any thread stole tasks from another worker thread",
+            min_steal_operations.clone(),
+        );
+
+        let num_remote_schedules = Gauge::default();
+        registry.register(
+            "nucliadb_node_num_remote",
+            "The number of tasks scheduled from outside the runtime. [...]",
+            num_remote_schedules.clone(),
+        );
+
+        let total_local_schedule_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_total_local_schedule_count",
+            "The number of tasks scheduled from worker threads. [...]",
+            total_local_schedule_count.clone(),
+        );
+
+        let max_local_schedule_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_max_local_schedule_count",
+            "The maximum number of tasks scheduled from any one worker thread",
+            max_local_schedule_count.clone(),
+        );
+
+        let min_local_schedule_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_min_local_schedule_count",
+            "The minimum number of tasks scheduled from any one worker thread",
+            min_local_schedule_count.clone(),
+        );
+
+        let total_overflow_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_total_overflow_count",
+            "The number of times worker threads staurated their local queues. [...]",
+            total_overflow_count.clone(),
+        );
+
+        let max_overflow_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_max_overflow_count",
+            "The maximum number of times any one worker saturated its local queue",
+            max_overflow_count.clone(),
+        );
+
+        let min_overflow_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_min_overflow_count",
+            "The minimum number of times any on worker saturated its local queue",
+            min_overflow_count.clone(),
+        );
+
+        let total_polls_count = Gauge::default();
+        registry.register(
+            "nucliadb_node_total_polls_count",
+            "The number of tasks that have been polled across all worker threads. [...]",
+            total_polls_count.clone(),
+        );
```

Comment:
Based on the description, this seems like only increases so maybe should be a `Counter`